### PR TITLE
RANCHER-1838 Create logic to install mod-linked-data and ui-ld-folio-wrapper on eureka-based environment

### PR DESCRIPTION
**Pull request by OHaimanov**
Jira: https://folio-org.atlassian.net/browse/RANCHER-1838

Tests:

- [createNamespaceFromBranch-Eureka-dev](https://jenkins-aws.indexdata.com/job/folioRancher/job/tmpFolderForDraftPipelines/job/OleksandrH/job/createNamespaceFromBranch-Eureka-dev/10/console)
- [uiBuild-Eureka-dev](https://jenkins-aws.indexdata.com/job/folioRancher/job/tmpFolderForDraftPipelines/job/OleksandrH/job/uiBuild-Eureka-dev/3/console)

### CODE REVIEW

**Comment by eldiiar-duishenaliev** [comment]
Is this something expected?
image

File: vars/folioUI.groovy [hunk 1]
```diff
@@ -1,11 +1,16 @@
 #!groovy
```
Comment (eldiiar-duishenaliev) [inline vars/folioUI.groovy:1]: are we sure about this lib? Seems like we're using this one: uiBuild.groovy...

Comment (OHaimanov) [reply vars/folioUI.groovy:1]: Yeap, folioUi it is refactored uiBuild. uiBuild will be removed soon

**Comment by OHaimanov** [comment]
> Is this something expected? image

Yeap, it looks like, module issue, linked data team will look into it

**Pull request merged**